ecs/service.go

```diff
@@ -336,3 +336,16 @@ func (ecs *ECS) RestartService(serviceName string) {
 		console.ErrorExit(err, "Could not restart service")
 	}
 }
+
+func (ecs *ECS) WaitUntilServiceStable(serviceName string) {
+	err := ecs.svc.WaitUntilServicesStable(
```

Comment:
Curious if there's a wait timeout?

Reply:
The default timeout seems to be 10 minutes. Couldn't find that documented but that's what it appears to be looking at the sdk source. There's a variant of that method that lets you specify timeout options -- https://docs.aws.amazon.com/sdk-for-go/api/service/ecs/#ECS.WaitUntilServicesStableWithContext 

```go
err := ecs.svc.WaitUntilServicesStableWithContext(
    aws.BackgroundContext(),
    &awsecs.DescribeServicesInput{
        Cluster:  aws.String(ecs.ClusterName),
        Services: aws.StringSlice([]string{serviceName}),
    },
    request.WithWaiterDelay(request.ConstantWaiterDelay(15*time.Second)),
    request.WithWaiterMaxAttempts(40),
)
```